docs/server-instructions.md

```diff
@@ -0,0 +1,102 @@
+# Add the JVM agent to Java servers
+
+The following sections show how to add the path to the Splunk OTel agent for Java using the [supported servers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#application-servers).
+
+## JBoss EAP / WildFly
+
+Add the `javaagent` argument to the `standalone` configuration file:
+
+- On Linux and macOS, add the following line at the end of the `standalone.conf` file:
+   ```
+   JAVA_OPTS="$JAVA_OPTS -javaagent:/path/to/splunk-otel-javaagent.jar"
+   ```
+- On Windows, dd the following line at the end of the `standalone.conf.bat` file:
+   ```
+   set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\splunk-otel-javaagent.jar"
+   ```
+
+## Jetty
+
+Configure the path to the JVM agent either in `start.ini` or in `jetty.sh`:
+
+- In the `start.ini` file, add the `javaagent` argument right below the `--exec` option:
+   ```
+   #===========================================================
+   # Sample Jetty start.ini file
+   #-----------------------------------------------------------
+   --exec
+   -javaagent:/path/to/splunk-otel-javaagent.jar
+   -Xmx512m
+   ...
+   ```
+-  In the `jetty.sh` file, add the `javaagent` argument inside `:
+   ```
+   JAVA_OPTIONS="${JAVA_OPTIONS} -javaagent\:/path/to/splunk-otel-javaagent.jar"
```

Comment:
```suggestion
   JAVA_OPTIONS="${JAVA_OPTIONS} -javaagent:/path/to/splunk-otel-javaagent.jar"
```
I tried it out, with the extra backslash it actually fails with `Unrecognized option: -javaagent\:/Users/..`